Gemfile

```diff
@@ -27,7 +27,7 @@ gem "stimulus-rails"
 gem "jbuilder"
 
 # Use Redis adapter to run Action Cable in production
-# gem "redis", ">= 4.0.1"
+gem "redis", ">= 4.0.1"
```

Comment:
Curious.. What is redis for?

Reply:
Redis its a gem that supports geocode gem but we don't need it anymore. I'll remove it.